client/state/selectors/is-gutenberg-opt-out-enabled.js

```diff
@@ -11,7 +11,7 @@ import isClassicEditorForced from 'state/selectors/is-classic-editor-forced';
 
 export const isGutenbergOptOutEnabled = ( state, siteId ) => {
 	return (
-		get( state, [ 'gutenbergOptIn', siteId ], true ) &&
+		get( state, [ 'gutenbergOptInOut', siteId, 'optOut' ], false ) &&
```

Comment:
We don't want to enable opt-out be default if there is a falsy response from the server. In fact, we want to disable opt-out if that's the case!